src/num.jl

```diff
@@ -170,12 +170,19 @@ tosymbolic(a::Num) = tosymbolic(value(a))
 tosymbolic(a::Sym) = Sym{symtype(a)}(nameof(a)) # unwrap stuff like Parameter{<:Number}
```

Comment:
I don't think we need this anymore. right?

Reply:
yeah